GitUI/UserControls/TextBoxEx.cs

```diff
@@ -60,24 +60,22 @@ protected override void WndProc(ref Message m)
             {
                 case NativeMethods.WM_NCPAINT:
                 case NativeMethods.WM_PAINT:
```

Comment:
❓ Can we not just set `UserPaint` style and move the draw code to `OnPaint`?

Reply:
This should be done in a dedicated PR on the main dev branch.